File: Telerik Algo Academy/Pull/Pull.cs
using System;

class Pull
{
    static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        double x1 = 0;
        double y1 = 0;
        for (int n = 0; n < N; n++)
        {
            string[] input = Console.ReadLine().Split(' ');
            double F = double.Parse(input[2]);
            x1 += F * double.Parse(input[0]);
            y1 += F * double.Parse(input[1]);
        }

        Console.WriteLine("{0} {1}", -x1, -y1);
    }
}

File: Telerik Algo Academy/Pull/Pull.js
function solve(args) {
    var N = args[0];
    var x1 = 0;
    var y1 = 0;
    for (var n = 0; n < N; n++) {
        var input = args[n + 1].split(' ');
        var F = input[2];
        x1 += F * input[0];
        y1 += F * input[1];
    }

    return -x1 + " " + -y1;
}